### PR TITLE
feat: moving libraries-bom to this repository

### PR DESCRIPTION
**Pull request by suztomo**
Moving the Libraries BOM from https://github.com/GoogleCloudPlatform/cloud-opensource-java repository.

### CODE REVIEW

File: .github/workflows/full-convergence-check.yaml [hunk 1]
```diff
@@ -19,5 +19,5 @@ jobs:
         mvn -B -V -ntp install
     - name: Validate dependency convergence of library dependencies in the Google Cloud BOM
       run: |
-        mvn -B -V -ntp validate
-      working-directory: dependency-convergence
+        mvn -B -V -ntp verify
+      working-directory: tests
```
Comment (suztomo) [inline .github/workflows/full-convergence-check.yaml:23]: BomContentText.java should run as part of this.

**Comment by suztomo** [comment]
I think I'll need to configure

**Comment by suztomo** [comment]
As per https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#bootstrap-with-cli, I'll need to run the CLI after merging this PR 4308.

File: libraries-bom/pom.xml [hunk 2]
```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>libraries-bom</artifactId>
```
Comment (suztomo) [inline libraries-bom/pom.xml:9]: This file is from https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/boms/cloud-oss-bom/pom.xml

File: libraries-bom/pom.xml [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+  <version>25.3.1-SNAPSHOT</version><!-- {x-version-update:libraries-bom:current} -->
```
Comment (suztomo) [inline libraries-bom/pom.xml:10]: Marker for the version managed by Release Please. A corresponding entry has been added to versions.txt.

File: libraries-bom/pom.xml [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+        <version>1.6.13</version>
+        <extensions>true</extensions>
+        <configuration>
+          <serverId>sonatype-nexus-staging</serverId>
+          <nexusUrl>https://google.oss.sonatype.org/</nexusUrl>
+          <autoReleaseAfterClose>false</autoReleaseAfterClose>
+        </configuration>
+      </plugin>
```
Comment (suztomo) [inline libraries-bom/pom.xml:302]: Added to follow google-cloud-bom.

**Review by Neenu1995 — APPROVED** [review]

File: libraries-bom/pom.xml [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+  <distributionManagement>
+    <snapshotRepository>
+      <id>sonatype-nexus-snapshots</id>
+      <url>https://google.oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>sonatype-nexus-staging</id>
+      <url>https://google.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+  </distributionManagement>
```
Comment (suztomo) [inline libraries-bom/pom.xml:75]: Added to follow google-cloud-bom.

File: libraries-bom/pom.xml [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+    <http.version>1.41.7</http.version>
+    <protobuf.version>3.19.4</protobuf.version>
+    <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
+        When updating gax.version, update gax.httpjson.version too. -->
+    <gax.version>2.16.0</gax.version>
+    <gax.httpjson.version>0.101.0</gax.httpjson.version>
+    <auth.version>1.6.0</auth.version>
+    <api-common.version>2.1.5</api-common.version>
+    <common.protos.version>2.8.3</common.protos.version>
+    <iam.protos.version>1.3.1</iam.protos.version>
```
Comment (suztomo) [inline libraries-bom/pom.xml:63]: I envision these properties will be gone once we start using the first-party-dependencies BOM (a part of the shared dependencies BOM). But for now I focus on the move of the pom to this repository.

The first-party-dependencies change will happen after the successful release of the Libraries BOM from this repository.

**Pull request merged**